event_types.go

```diff
@@ -595,6 +595,10 @@ type MergeEvent struct {
 			Previous int `json:"previous"`
 			Current  int `json:"current"`
 		} `json:"updated_by_id"`
+		Title struct {
+			Previous string `json:"previous"`
+			Current  string `json:"current"`
+		}
```

Comment:
Please add a `json` tag:
```suggestion
		} `json:"title"`
```

Reply:
fixed.